### ISSUE TITLE
Infinite red error box when hovering over any doorknob

### ISSUE THREAD

**Issue by TortuousAugur** [blind-coder/pz-dihakft#3]
I've located the log files but couldn't figure out which file it was. I know I had some *really* old logs in there so I deleted them all. The error is super easy to recreate anyway.

I'm not sure if it's a keyed or unkeyed doorknob (I don't get a tooltip) but when I hover over any doorknob I've collected I get a red error box that increases it's number very rapidly. I've copied the repeating segment from the console window that runs conjunction to PZ when I hover over a doorknob. Please let me know if there is anything else I can do to further clarify and locate the issue.

```
1593433517723
1593433517724 -----------------------------------------
1593433517724 STACK TRACE
1593433517724 -----------------------------------------
1593433517724 function: BitAND -- file: dihakft.lua line # 5
1593433517724 function: ID2Text -- file: dihakft.lua line # 57
1593433517724 function: render -- file: dihakft.lua line # 82
java.lang.RuntimeException: __lt not defined for operand
        at se.krka.kahlua.vm.KahluaUtil.fail(KahluaUtil.java:83)
        at se.krka.kahlua.vm.KahluaThread.luaMainloop(KahluaThread.java:902)
        at se.krka.kahlua.vm.KahluaThread.call(KahluaThread.java:167)
        at se.krka.kahlua.vm.KahluaThread.pcall(KahluaThread.java:1922)
        at se.krka.kahlua.vm.KahluaThread.pcallvoid(KahluaThread.java:1792)
        at se.krka.kahlua.integration.LuaCaller.pcallvoid(LuaCaller.java:38)
        at zombie.ui.UIElement.render(UIElement.java:1264)
        at zombie.ui.UIManager.render(UIManager.java:707)
        at zombie.gameStates.IngameState.renderframeui(IngameState.java:960)
        at zombie.gameStates.IngameState.render(IngameState.java:1109)
        at zombie.gameStates.GameStateMachine.render(GameStateMachine.java:37)
        at zombie.GameWindow.render(GameWindow.java:745)
        at zombie.GameWindow.run(GameWindow.java:1323)
        at zombie.GameWindow.maina(GameWindow.java:1091)
        at zombie.gameStates.MainScreenState.main(MainScreenState.java:184)
1593433517725
1593433517726 -----------------------------------------
1593433517726 STACK TRACE
1593433517726 -----------------------------------------
1593433517726 function: BitAND -- file: dihakft.lua line # 5
1593433517726 function: ID2Text -- file: dihakft.lua line # 57
1593433517726 function: render -- file: dihakft.lua line # 82
```

**Comment by TortuousAugur** [comment]
Forgot to mention, I also get no tooltip when hovering over a doorknob.

**Comment by blind-coder** [comment]
Fixed in v3.1.2
